src/internal/m365/collection/drive/helper_test.go

```diff
@@ -996,6 +1022,78 @@ func (dd *deltaDrive) newEnumer() *DeltaDriveEnumerator {
 	return &DeltaDriveEnumerator{Drive: clone}
 }
 
+type drivePrevPaths struct {
+	id                 string
+	folderIDToPrevPath map[string]string
+}
+
+func (dd *deltaDrive) newPrevPaths(
+	idPathPairs ...string,
+) *drivePrevPaths {
+	dpp := drivePrevPaths{
+		id:                 dd.id,
+		folderIDToPrevPath: map[string]string{},
+	}
+
+	if len(idPathPairs)%2 == 1 {
+		dpp.folderIDToPrevPath["error"] = "idPathPairs had odd count of elements"
```

Comment:
since this is a new function I'd prefer if it took `*testing.T` and used `require.Zero` or something like that. This seems like it could be error prone because not everyone may know they need to check for a specific key in the map

Reply:
Heh, okay.